src/components/Post/PostDetailComponent.jsx

```diff
@@ -1,16 +1,128 @@
 import React, { Component } from 'react';
-import { Segment } from 'semantic-ui-react';
+import { Segment, Image } from 'semantic-ui-react';
+import { withRouter, Link } from 'react-router-dom';
+import GoogleMapReact from 'google-map-react';
+
+import MapMarker from '../Home/MapMarkers';
 
 class NewPost extends Component {
+  state = {
+    zoom:15,
+  }
+  
+  postColorHandler = (post) => {
+    let postColor = '';
+    if (post.user_rating === 'Gnarly') {
+      postColor = 'red';
+    } else if (post.user_rating === 'Good') {
+      postColor = 'orange';
+    } else if (post.user_rating === 'Fair') {
+      postColor = 'yellow';
+    } else if (post.user_rating === 'Poor') {
+      postColor = 'teal';
+    } else if (post.user_rating === 'Flat') {
+      postColor = 'grey';
+    }
+    return postColor;
+  };
+  
+  getPost = async () => {
+    if (this.state.post === undefined) {
+      const response = await fetch(`${process.env.REACT_APP_DEV_API_DOMAIN}/posts/${this.props.match.params.post_id.toString()}`);
```

Comment:
You don't have to apply `.toString()` because string interpolation automatically turns this into a string when concatenated.